examples/mugen/README.md

```diff
@@ -0,0 +1,13 @@
+# MUGEN: A Playground for Video-Audio-Text Multimodal Understanding and GENeration
+
+Implementation of [MUGEN](https://arxiv.org/abs/2204.08058) in TorchMultimodal for text-video [retrieval](https://github.com/facebookresearch/multimodal/tree/main/examples/mugen/retrieval) and [generation](https://github.com/facebookresearch/multimodal/tree/main/examples/mugen/generation).
```

Comment:
nit: would make it explicit here that more detailed instructions for each of the two tasks can be found in the linked directories.